docker/Dockerfile

```diff
@@ -8,10 +8,19 @@ ENV PATH $CASSANDRA_DIR/bin:$PATH
 
 # Untar our distribution
 ADD palantir-cassandra-*tgz /
+RUN mkdir $CASSANDRA_DIR/logs; \
+  mkdir -p $CASSANDRA_DIR/data/saved_caches; \
+  mkdir -p /var/lib/cassandra/persistent_settings; \
```

Comment:
This is super lame, not sure why we cannot just use the default directory path (which should be `$CASSANDRA_HOME/data`, and the folders being owned by the `cassandra` user). For whatever reason, on the new version, we're failing to create these directories, so I do wonder if this never worked. Opt'd instead just to create directories manually, which isn't the craziest idea (the C* 3.X does something similar)